app.ts

```diff
@@ -31,4 +31,5 @@ app.listen(3000, () => {
 });
 
 require('./controller/HelloWorldController')(app);
```

Comment:
You can remove all references to `HelloWorldController` and it's supporting code